content/en/docs/setup/platform-setup/k3d/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: k3d
+description: Instructions to set up k3d for Istio.
+weight: 30
+skip_seealso: true
+keywords: [platform-setup,kubernetes,k3d,k3s]
+owner: 
+test: no
+---
+
+k3d is a lightweight wrapper to run [k3s](https://github.com/rancher/k3s) (Rancher Lab’s minimal Kubernetes distribution) in docker.
+
+k3d makes it very easy to create single- and multi-node k3s clusters in docker, e.g. for local development on Kubernetes.
+
+## Prerequisites
+
+- To use k3d, you will also need to [install docker](https://docs.docker.com/install/).
+- Install the latest version of [k3d](https://k3d.io/v5.4.7/#installation).
+- To interact with the Kubernetes cluster [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- (Optional) [Helm](https://helm.sh/docs/intro/install/) is the package manager for Kubernetes
+
+## Installation steps
+
+1.  Create a cluster and disable Traefik with the following command:
+
+    {{< text bash >}}
+    $ k3d cluster create --api-port 6550 -p "9080:80@loadbalancer"  -p "9443:443@loadbalancer" --agents 2 --k3s-arg '--disable=traefik@server:*'
+    {{< /text >}}
+
+1.  To see the list of k3d clusters, use the following command:
+
+    {{< text bash >}}
+    $ k3d cluster list
+    k3s-default
+    {{< /text >}}
+
+1.  To list the local Kubernetes contexts, use the following command.
+
+    {{< text bash >}}
+    $ kubectl config get-contexts
+    CURRENT   NAME                 CLUSTER              AUTHINFO             NAMESPACE
+    *         k3d-k3s-default      k3d-k3s-default      k3d-k3s-default
+    {{< /text >}}
+
+    {{< tip >}}
+    `k3d-` is prefixed to the context and cluster names, for example: `k3d-k3s-default`
+    {{< /tip >}}
+
+1.  If you run multiple clusters, you need to choose which cluster `kubectl` talks to. You can set a default cluster
+    for `kubectl` by setting the current context in the [Kubernetes kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file. Additionally you can run following command
+    to set the current context for `kubectl`.
+
+    {{< text bash >}}
+    $ kubectl config use-context k3d-k3s-default
+    Switched to context "k3d-k3s-default".
+    {{< /text >}}
+
+## Setup Istio for K3d
```

Comment:
```suggestion
## Set up Istio for K3d

```
Provide a blank line after the heading to avoid lint errors.

Reply:
@windsonsea fixed d12cc24fc105e41ea199fc305772d9e1fa204387